src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -532,39 +591,15 @@ impl HummockVersionUpdateExt for HummockVersion {
                 version_delta.max_committed_epoch,
                 self.max_committed_epoch
             );
+
             if self.max_committed_epoch < version_delta.max_committed_epoch {
                 // `max_committed_epoch` increases. It must be a `commit_epoch`
```

Comment:
Please help to fix the comment: It must be a `commit_epoch` or compaction group metadata change.